src/main/java/uk/gov/ons/ctp/response/sample/endpoint/SampleEndpoint.java

```diff
@@ -131,10 +135,13 @@ public SampleSummary ingest(final MultipartFile file, final String type) throws
   }
 
   @RequestMapping(value = "/{type}/fileupload", method = RequestMethod.POST, consumes = "multipart/form-data")
-  public final @ResponseBody ResponseEntity<SampleSummary> uploadSampleFile(@PathVariable("type") final String type, @RequestParam("file") MultipartFile file) throws CTPException {
+  public final @ResponseBody
+  ResponseEntity<SampleSummary> uploadSampleFile(@PathVariable("type") final String type, @RequestParam("file") MultipartFile file) throws CTPException {
     log.info("Entering Sample file upload for Type {}", type);
-    if (!Arrays.asList("B", "CENSUS", "SOCIAL").contains(type.toUpperCase())) {
-      return ResponseEntity.badRequest().build();
+    if (!Arrays.asList("B", "CENSUS", "SOCIAL")
```

Comment:
Is there any reason why its called `B` rather than `Business`?

Reply:
Not that I can see - it was originally built like that.   If we change it now we'll need to change it in response-operations-ui etc

Reply:
No thats fine I was just curious more than anything else 👍